.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,75 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/go
+{
+	"name": "Go",
+	"build": {
+		"dockerfile": "DockerfileDev",
+		"args": {
+			"USERNAME": "${localEnv:USER}",
+			"USER_UID": "${localEnv:UID}",
+			"USER_GID": "${localEnv:UID}",
+			"MAIL": "${localEnv:MAIL}",
+			"FULLNAME": "${localEnv:FULLNAME}"
+		}
+	},
+	"initializeCommand": "touch .env",
```

Comment:
You should probably add .env to .gitignore just to be safe

Reply:
It is already added to .gitignore (line 25) :)